ca/config.go

```diff
@@ -135,6 +135,13 @@ func (s *SecurityConfig) UpdateRootCA(cert, key []byte, certExpiry time.Duration
 	return err
 }
 
+// loadNewRootCA replaces the root CA with a new root CA
+func (s *SecurityConfig) loadNewRootCA(rootCA *RootCA) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	s.rootCA = rootCA
+}
```

Comment:
> Perhaps we need to make sure that when a node becomes a leader (i.e. when ca.Server starts), the root CA on disk and in the security config is sync'd with the cert/key in Raft.

I believe it is because the ca server starts a process whereby raft store is watched for changes, and any changes to the cluster (and the initial watch itself) cause the raft CA cert and key to be loaded into the security config.  

Non leaders though do not update their certs based on raft, however, and currently seem to depend only on what is on disk and what might come as a result of updating heir TLS certs.